### ISSUE TITLE
Image preprocessing

### ISSUE THREAD

**Issue by nawazia** [BeileiCui/SurgicalDINO#12]
Hi,

I'm trying to assess Surgical-DINO on StereoMIS & natural images and not sure how to preprocess the images (the rendered depth looks incorrect)? Currently doing:

> transform = transforms.Compose([
>         transforms.Resize(image_shape),
>         transforms.ToTensor(),
>         transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
> ])
> image = Image.open("path/to/image.png").convert("RGB")
> original_size = image.size[::-1]  # (H, W)
> test = transform(image).unsqueeze(0).to(device)
> output = surgicaldino(pixel_values=test)

Thanks

**Comment by BeileiCui** [comment]
@nawazia Hi, I remember that, depending on the type of foundation model weight used, the size of input should be a multiple of 14 (cause the patch size of dinov2 is 14). Also, we did not provide pretrained weights for SurgicalDINO, so you have to train it by yourself. A more advanced version, EndoDAC provide pretrained weights and training scripts; you can check on that one.
